internal/lib/lib.go

```diff
@@ -1725,3 +1727,18 @@ func GetThrottle(key string) *int32 {
 	throttle := int32(throttle[key])
 	return &throttle
 }
+
+func UpdateV6(vip *avimodels.Vip, vipNetwork *akov1alpha1.AviInfraSettingVipNetwork) {
+	v6addr := strings.Split(vipNetwork.V6Cidr, "/")[0]
+	v6mask, _ := strconv.Atoi(strings.Split(vipNetwork.V6Cidr, "/")[1])
+
+	v6cidr := avimodels.IPAddrPrefix{
+		IPAddr: &avimodels.IPAddr{
+			Addr: &v6addr,
+			Type: proto.String("V6"),
+		},
+		Mask: proto.Int32(int32(v6mask)),
+	}
+	vip.AutoAllocateIPType = proto.String("V4_V6")
```

Comment:
`AutoAllocateIPType`  has some license based restrictions. Can you please check that?

Reply:
this will be documented